test/ui-test/testSuites/suite_wallet/tst_wallet_accountsManagement/test.feature

```diff
@@ -15,7 +15,7 @@ Feature: Status Desktop Wallet Section Wallet Account Management
         Then the account is correctly displayed with "<new_name>" and "#<new_color>" and emoji unicode "<new_emoji_unicode>" in accounts list
         Examples:
             | name           | new_name         | new_color | new_emoji  | new_emoji_unicode |
-            | Status account | MyPrimaryAccount | 216266    | sunglasses | 1f60e             |
+            | Account 1 | MyPrimaryAccount | 216266    | sunglasses | 1f60e             |
```

Comment:
@saledjenic thank you for ignoring me during the presentation where i was saying that there is no longer need to support OLD tests :)